docs/what-is-datahub/datahub-concepts.md

```diff
@@ -0,0 +1,111 @@
+# DataHub Concepts
+
+Explore key concepts of DataHub to take full advantage of its capabilities in managing your data.
+
+## General Concepts
+
+### URN (Uniform Resource Name)
+URN (Uniform Resource Name) is the chosen scheme of URI to uniquely define any resource in DataHub. It has the following form.
+```
+urn:<Namespace>:<Entity Type>:<ID>
+```
+> * [What is URN?](/docs/what/urn.md)
+
+### Entity
+An entity is the primary node in the metadata graph. For example, an instance of a Dataset or a CorpUser is an Entity. 
+
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+### Aspect
+An aspect is a collection of attributes that describes a particular facet of an entity. 
+Aspects can be shared across entities, for example "Ownership" is an aspect that is re-used across all the Entities that have owners. 
+
+> * [What is a metadata aspect?](/docs/what/aspect.md)
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+### Policy
+Access policies in DataHub define who can do what to which resources. 
+
+> * [Policies Guide](/docs/authorization/policies.md)
+> * [Developer Guides: DataHubPolicy](/docs/generated/metamodel/entities/dataHubPolicy.md)
+> * [Feature Guides: About DataHub Access Policies](/docs/authorization/access-policies-guide.md)
+
+### Role
+DataHub provides the ability to use Roles to manage permissions.
+
+> * [About DataHub Roles](/docs/authorization/roles.md)
+> * [Developer Guides: DataHubRole](/docs/generated/metamodel/entities/dataHubRole.md)
+
+
+### Relationships 
+A relationship represents a named edge between 2 entities. They are declared via foreign key attributes within Aspects along with a custom annotation (@Relationship). 
+
+> * [What is a relationship?](/docs/what/relationship.md)
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+## Metadata Models
```

Comment:
I would say let's not have a section called "Metadata models". 

Most users don't think of these as "Metadata Models" they think of them as concepts or features provided by the DataHub UI. 

Only admins and engineers will understand what is meant by a "Metadata Model" -- even they might be confused actually.

We can have a section at the very bottom calledd "Metadata Model" which includes _only_ Entity, Aspect, Relationship overviews and points to another doc which goes into more depth about this concept.